Updates Analysis

bindalraghav opened the pull request:
State-wise Analysis of India.

bindalraghav committed:
updated covid_trend_analysis.py

bindalraghav committed:
Updated analysis of India

State-wise analysis of India

bindalraghav committed:
Updated India Analysis

State-wise Analysis

bindalraghav committed:
India Analysis Updated

manojgoyal04 reviewed (changes requested):
Please resolve merge conflicts

bindalraghav committed:
Merge branch 'master' into master

bindalraghav committed:
Update covid_trend_analysis.py

bindalraghav committed:
Update covid_trend_analysis.py

bindalraghav committed:
Glimpse of project.

By this presentation, I  tried to explain my project and show some of my conclusions according to my analysis and visualizations.

bindalraghav committed:
Updated Read.me
